git_patrol.py

```diff
@@ -159,30 +162,38 @@ def __init__(self):
   # Provide a few default substitutions that Google Cloud Build would fill in
   # if it was launching a triggered workflow. See link for details...
   # https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values
-  arg_substitutions = '--substitutions='
+  substitutions_list = []
   if git_ref.startswith('refs/tags/'):
-    arg_substitutions += 'TAG_NAME={},'.format(
-        git_ref.replace('refs/tags/', ''))
+    substitutions_list.append(
+        'TAG_NAME={}'.format(git_ref.replace('refs/tags/', '')))
   elif git_ref.startswith('refs/heads/'):
-    arg_substitutions += 'BRANCH_NAME={},'.format(
-        git_ref.replace('refs/heads/', ''))
+    substitutions_list.append(
+        'BRANCH_NAME={}'.format(git_ref.replace('refs/heads/', '')))
```

Comment:
this could be DRYed up a bit if you want

Reply:
Can do. What were you thinking?